cpp/include/Ice/Instrumentation.h

```diff
@@ -1049,7 +369,7 @@ class ICE_API CommunicatorObserver : public virtual ::Ice::LocalObject
      * @param ctx The context specified by the user.
      * @return The invocation observer to instrument the invocation.
      */
-    virtual InvocationObserverPtr getInvocationObserver(const ::Ice::ObjectPrx& prx, const ::std::string& operation, const ::Ice::Context& ctx) = 0;
+    virtual InvocationObserverPtr getInvocationObserver(const Ice::ObjectPrxPtr& prx, const ::std::string& operation, const ::Ice::Context& ctx) = 0;
```

Comment:
Is this change from `ObjectPrx` to `ObjectPrxPtr` intentional?
It doesn't actually matter because I believe these are type-defs of each other, but still.
Originally it was just a plain `ObjectPrx`.

Reply:
Yes, it's to use the same code with both mappings.